src/tasks/MonoTargetsTasks/ILStrip/ILStrip.cs

```diff
@@ -351,17 +350,15 @@ private static int ComputeMethodHeaderSize(MemoryStream memStream, int actualLoc
         return (headerFlag == 2 ? 1 : 4);
     }
 
-    private static void SetCodeSizeToZeroForTiny(ref MemoryStream memStream, int actualLoc)
+    private static void SetCodeSizeToZeroForTiny(MemoryStream memStream, int actualLoc)
     {
         memStream.Position = actualLoc;
-        byte[] header = {0b10};
-        memStream.Write(header, 0, 1);
+        memStream.WriteByte(0b10);
     }
 
-    private static void ZeroOutMethodBody(ref MemoryStream memStream, int methodSize, int actualLoc, int headerSize)
+    private static void ZeroOutMethodBody(MemoryStream memStream, int methodSize, int actualLoc, int headerSize)
```

Comment:
Mapping the region as `Span` and then `Span.Clear` will be cheaper than renting new buffer + copy.

Reply:
I might be missing something but why are we using a copied MemoryStream at all rather than just directly writing to the FileStream?